docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3"
```

Comment:
Shouldn't this be in the `tests/Application` directory to fully work with the Symfony binary? Also I have never added `docker-compose.yaml` files to my plugins, but I am not against it per se - there should just be a section in the `README.md` about it then.